src/main/java/yevhenii/lostfilmdemo/services/impl/SchedulerServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package yevhenii.lostfilmdemo.services.impl;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.quartz.*;
+import org.springframework.stereotype.Service;
+import yevhenii.lostfilmdemo.quartz.LostfilmRssJob;
+import yevhenii.lostfilmdemo.services.SchedulerService;
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+import static org.quartz.TriggerKey.triggerKey;
+
+@Log4j2
+@Service
+@RequiredArgsConstructor
+public class SchedulerServiceImpl implements SchedulerService {
+    @Getter
+    static int jobCount;
+    @Getter
+    static int triggerCount;
+    private final Scheduler scheduler;
+
+    public void start() throws SchedulerException {
+
+        Trigger trigger = getTrigger(getJobDetail());
+        if (!scheduler.checkExists(trigger.getKey()))
+        scheduler.scheduleJob(trigger);
```

Comment:
you don't need this as well.

https://github.com/kenshoo/fb-dca-template-manager/blob/master/application/src/main/java/io/skai/fb/dca/template/quartz/config/JobDetailsCfg.java

https://github.com/kenshoo/fb-dca-template-manager/blob/master/application/src/main/java/io/skai/fb/dca/template/quartz/job/ExampleJob.java